packages/widget-chat/src/actions/conversation.js

```diff
@@ -58,6 +66,9 @@ export function listenToMercuryActivity(conversationId, spark) {
     spark.mercury.on(`event:conversation.activity`, (event) => {
       const activity = event.data.activity;
       if (activity.object.objectType === `comment` && activity.target.id === conversationId) {
+        dispatch(receiveMercuryComment(activity));
+      }
+      else if (activity.object.objectType === `activity` && activity.target.id === conversationId) {
```

Comment:
yea, it's not obvious it's necessary, but if we don't add the explicit checks now, we're going to end up with weird breakages in the future. the idea is basically: check all conditions and if there's no match, discard the activity. better to call attention to not-yet-implemented features that break catastrophically (which has happened when verbs get overloaded).